package it.polito.ai.laboratorio3.exceptions;

public class TeamNameAlreadyInCourseExceptions extends TeamServiceException {
    public TeamNameAlreadyInCourseExceptions(){

        super("The team name is already used! ");
} }
